presets/csscomb-preset-unity/README.md

```diff
@@ -0,0 +1,16 @@
+# csscomb-preset-unity
+
+Csscomb preset.
+
+## Usage
+
+### Install:
+
+```bash
+npm install --save-dev csscomb-preset-unity
+```
+
+### Configure
+
+After install `csscomb-preset-unity` will copy `.csscomb.json` to your project in the root directory with package.json.
```

Comment:
After installlation `csscomb-preset-unity` will copy `.csscomb.json` to your project's root directory containing package.json.
  If you want, you can override or change configs in that `.csscomb.json`. If you already have a `.csscomb.json` file, it will be overridden by `csscomb-preset-unity`.